app/controllers/likes_controller.rb

```diff
@@ -1,45 +1,44 @@
 class LikesController < ApplicationController
   before_action :authenticate_user!
+  before_action :find_like, only: [:destroy]
+
 
 
-=begin
   def create
+    session[:return_to] ||= request.referer
     @post = Post.find(params[:post_id])
+    @post.likes.create(user: current_user, post: @post)
+    @like = @post.likes.where(user: current_user, post: @post)[0]
```

Comment:
You can use the method `first` here. That would be better. 
This code will break in case array has no element.

Reply:
Done